### PR TITLE
add document for device field in networkchaos

### PR DESCRIPTION
**Pull request by YangKeao**

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-network-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ NetworkChaos 用于模拟集群中网络故障的场景，目前支持以下几
 | containerNames | []string | 指定注入的容器名称 | 无 | 否 | ["nginx"] |
 | selector | struct | 指定注入故障的目标 Pod，详情请参考[定义实验范围](./define-chaos-experiment-scope.md) | 无 | 是 |  |
 | externalTargets | []string | 表示 Kubernetes 之外的网络目标, 可以是 IPv4 地址或者域名。只能与 `direction: to` 一起工作。 | 无 | 否 | 1.1.1.1, www.google.com |
+| device | string | 指定影响的网络设备 | 无 | 否 | eth0 |
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-network-chaos-on-kubernetes.md:149]: 这里的 eth0 需要加个标点符号吗？之前在 “simulate-network-chaos-in-physical-nodes.md " 看到对应内容统一加上了

```suggestion
| device | string | 指定影响的网络设备 | 无 | 否 | "eth0" |
```

**Comment by en-jin19** [comment]
@YangKeao could you please invite a tech reviewer? thanks!

**Review by Hexilee — APPROVED** [review]
LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-network-chaos-on-kubernetes.md (hunk 1 above)
Comment (YangKeao) [reply i18n/zh/docusaurus-plugin-content-docs/current/simulate-network-chaos-on-kubernetes.md]: Fixed.

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**